docs/content/service/web/settings.md

```diff
@@ -16,33 +18,71 @@ Use `.docksal/etc/apache/httpd-vhosts.conf` to define additional virtual hosts:
 
 ```apacheconfig
 <VirtualHost *:80>
-	ServerName docs.test.docksal
-
-	ProxyPass / http://docs.docksal.io/
-	ProxyPassReverse / http://docs.docksal.io/
+    ServerName ${APACHE_SERVERNAME}
+    ServerAlias styleguide.*
+    DocumentRoot /var/www/styleguide
 </VirtualHost>
+
+<Directory "/var/www/styleguide">
+    Require all granted
+</Directory>
 ```
 
-## Using Different Apache Versions {#apache-versions}
+For a complete list of available features and settings see [docksal/service-apache](https://github.com/docksal/service-apache).
+
+
+### Nginx Advanced Settings {#nginx-settings}
+
+The following settings files in the project codebase can be used to override default Apache settings.
```

Comment:
Should this be Nginx settings, not Apache?

Reply:
`service/web/settings.md` now covers both Apache and Nginx. Updated title.